hw2/style-hw2.css

```diff
@@ -0,0 +1,204 @@
+* {
+	padding: 0;
+	margin: 0;
+}
+
+body {
+	background-color: #f9f9f9;
+	font-family: Geneva, Arial, Helvetica, sans-serif;
+}
+
+a {
+	text-decoration: none;
+}
+/*----------header----------*/
+.container {
+	width: 75%;
+	background-color: #fff;
+	margin: 0 auto;
+	box-shadow: 0 0 3px 2px #dbdbdb;
+	padding: 50px 45px 60px 45px;
+}
+
+.main-menu {
+	margin-top: 35px;
+	height: 40px;
+	background: linear-gradient(to top, #444343, #5e5e5e);
+}
+
+.main-menu__item {
+	list-style: none;
+	float: left; 
+	width: 13%;
+	border-right: rgba(255,255,255,0) 1px solid;
+	border-left: rgba(255,255,255,0) 1px solid;
+}
+
+.main-menu__link {
+	color: #ffffff;
+	display: block;
+	height: 100%;
+	text-align: center;
+	padding: 10px;	
+} 
+
+.main-menu__item:hover {
+	border-right: rgba(255,255,255,1) 1px solid;
+	border-left: rgba(255,255,255,1) 1px solid;
+	background: linear-gradient(to top, #2f2f2f, #4a4a4a);
+}
+/*----------middle-part----------*/
+.middle-container {
+	display: flex;
+	align-items: flex-start;
+	justify-content: space-between;
+	padding-top: 40px;
+}
+
+.products {
+	width: 25%;
+	display: inline-block;
+	border: 1px solid #c8c8c8;
+}
+
+
+.products__item {
+	list-style: none;
+	border-bottom: 1px solid #c8c8c8;
+	color: #807e7e;
+}
+
+.products__item:last-child {
+	border-bottom: none;
+}
+
+.products__link{
+	color: #807e7e;
+	display: block;
+	height: 100%;
+	padding: 15px 25px;
+} 
+
+.products__item:hover {
+	background-color: #f9f9f9;
+} 
+
+.products__item:hover .products__link {
+	color: #000;
+	padding-left: 35px;
+} 
+
+.main-info {
+	width: 70%;
+}
+
+.main-info__item {
+	border: 1px solid #c8c8c8;
+	height: 400px;
+	display: flex;
+	justify-content: center;
+	align-items: center;	
+}
+
+.main-info__text {
+	padding: 35px 25px 60px 25px;
+	color: #2c2c2c;
+	line-height: 150%;
+}
+
+/*----------footer----------*/
+.main-footer {
+	height: 40px;
+	background-color: #e1e1e1;
+}
+
+.footer-menu__item {
+	list-style: none;
+	display: block;
+	float: left;
+	width: 13%;
+	
+}
+
+.footer-menu__link {
+	color: #2c2c2c;
+	font-size: 90%;
+	display: block;
+	height: 100%;
+	text-align: center;
+	padding: 10px; 
+}
+
+.footer-menu__item:hover .footer-menu__link  {
```

Comment:
link and item are pretty the same (by size), so you can  omit including selectors here